Dockerfile

```diff
@@ -38,6 +40,7 @@ RUN dnf install --installroot /tmp/rootfs-dependency \
       /tmp/rootfs-dependency/var/log/dnf* \
       /tmp/rootfs-dependency/var/log/yum.*
 
+# platform is required, otherwise the copy command will copy the wrong architecture files
```

Comment:
is it wrong place for comment? I assume we should move before line 2?

Reply:
removed it, there was a `--platform` there previously but the Github CI was showing warnings saying "this is the default behaviour you dummy" so I removed it, but not the comment :D